### PR TITLE
Create new navigator

### PR DESCRIPTION
**Pull request by civing**
fix Lundalogik/crm-feature#2688

### CODE REVIEW

**Review by jgroth — COMMENTED** [review]
Looks good to me. We need to improve the documentation, so if we are not doing it in this PR we need to create a new issue to fix that

**Comment by maltja** [comment]
> Looks good to me. We need to improve the documentation, so if we are not doing it in this PR we need to create a new issue to fix that

fixed 🙂

File: packages/interfaces/src/service/navigation-service.interface.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+ * @experimental
+ */
+export interface Navigator {
+    /**
+     * Navigate to a url
+     *
+     * @param {string} url the url string
+     */
+    navigate(url: string): void;
+}
```
Comment (jgroth) [inline packages/interfaces/src/service/navigation-service.interface.ts:13]: We should probably also add a type/interface for the event that gets dispatched from this service after navigating

Comment (maltja) [reply packages/interfaces/src/service/navigation-service.interface.ts:14]: fixed, added type Location. 🙂

**Review by jgroth — DISMISSED** [review]
Looks good! I think we need to work on the documentation a bit, but let's create a new issue for that so we can get this merged :slightly_smiling_face:

**Review by jgroth — APPROVED** [review]

**Review by jgroth — APPROVED** [review]

**Pull request merged**